lib/js/src/protocol/SdlProtocolBase.js

```diff
@@ -414,20 +414,6 @@ class SdlProtocolBase {
      * @param {SdlPacket} sdlPacket
      */
     _handleStartServiceNAK (sdlPacket) {
```

Comment:
Is this meant to be a reflection of Java Suite's `handleProtocolSessionNAKed`?

Reply:
Yes, the functions called service here were protocolSession in the java sdk. Is there a listener that should be called here? It was calling SessionEndedNACKed incorrectly and that was removed but it seems like it should call something.

Reply:
If it did it seems like it should be in SdlSession. There is an onProtocolSessionEndedNACKed, onProtocolSessionStarted, but no onProtocolSessionStartedNACKed.

Reply:
Looks like the Java Suite calls `onProtocolSessionNACKed`, which we don't have defined in our `SdlProtocolListener` class and likely needs to be added.

Reply:
I've added listeners for onProtocolSessionNACKed and onProtocolError to SdlProtocolListener. Nothing is hooked up to these at the moment but they are being called from SdlPrtoocolBase now.